src/Xamarin.iOS.ResourceDesigner/GenerateResourceDesignerTask.cs

```diff
@@ -10,6 +10,9 @@ public class GenerateResourceDesignerTask : Task
     {
         [Required]
         public ITaskItem[]? ImageAssets { get; set; }
+        
+        [Required]
+        public ITaskItem[]? ColorAssets { get; set; }
```

Comment:
Shouldn't it be renamed from ImageAssets to just Assets then? Yeah, I know that the resource tag is called ImageAsset, but it really looks like the wrong naming (as color assets was introduced in XCode 9, so I suppose there was no color assets when this tag was created) and "ImageAssets" name might be confusing for code readers. Then use Asset for anything ImageAsset tag-related (like AssetRawDto instead of ImageAsset...)?

Reply:
In this case we can continue use ColorAsset and ImageAsset naming, without renaming it to -Sets

Reply:
ImageAsset naming still remains ambiguous, since in csproj this node is like this. Probably you're right that it is a wrong naming in csproj files, but we will have it till if microsoft will rename this node

Reply:
I think we can do the following:
1. Call node `Assets`.  It will take as ITaskItem[] list of @(ImageAsset), since it is a legacy of Microsoft and Xamarin. But the name will clearly declare that they're assets in general.
2. In all occurences, rename `ImageAsset` into `ImageSet`. This will remove ambiguity with more general `ImageAsset` node in csproj files, declaring that these properties refer only to *.imageset items (iOS resources all image sets Contents.json files lays inside of *.imageset folders).
3. In all occurences, rename `ColorAsset` into `ColorSet`. This will have the same logic of naming as with `ImageSet`, but here declaring that these properties refer only to *.colorset items (iOS resources all color sets Contents.json files lays inside of *.colorset folders).